app/main.py

```diff
@@ -2,0 +1,54 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name, health: int = 100, is_hidden: bool = False):
+        self.name = name
+        self.health = health
+        self.is_hidden = is_hidden
+        self.alive.append(self)
```

Comment:
Appending `self` to `self.alive` is incorrect because `alive` should be accessed as a class variable. Use `Animal.alive.append(self)` instead.